faidx.c

```diff
@@ -360,55 +435,105 @@ int faidx_core(int argc, char *argv[], enum fai_format_options format)
         else
             fprintf(stderr, "\n");
 
-        return EXIT_FAILURE;
+        goto exit2;
     }
 
     /** output file provided by user */
     if( output_file != NULL ) {
         if( strcmp( output_file, argv[optind] ) == 0 ) {
             fprintf(stderr,"[faidx] Same input/output : %s\n", output_file);
-            return EXIT_FAILURE;
+            goto exit2;
+        }
+        if (!out.isbgzip) {
+            out.fp = fopen( output_file, "w" );
+        }
+        else {
```

Comment:
I know we don't have a style guide, but checking the source I see 21

```
}
else {
```

style clauses (6 of which are in reset.c) and 408

```
} else {
```

So basically 95% of the time we're keeping both braces on the same line.  Maybe we should go with the flow and try to keep a consistent bracing style where possible.


Don't even get me started on

```
if (expr)
{
```

Lol.  (It's mostly stats.c in samtools and VCF world in htslib, but it's needless whitespace in my book!)